install.ps1

```diff
@@ -101,6 +100,15 @@ function Get-DownloadUrl {
     return "https://dl.dagger.io/dagger/releases/${DaggerVersion}/${fileName}"
 }
 
+function Get-ChecksumUrl {
+
```

Comment:
nit: newlines :eyes: (sorry, bit of a broken record)

Would be totally happy if we could have automated linting checks for this script if you like, like I'm doing in #7633 for `install.sh`.

Reply:
no problem. Will fix this evening